frontend/src/Components/ValidityEvidence.tsx

```diff
@@ -6,17 +6,29 @@ import Tooltip from './Tooltip';
 type ValidityEvidence = CriiptoJwtSignature['validity'];
 type DrawableEvidence = CriiptoDrawableSignature['validity'];
 
-export default function ValidityChecks(props: { validityChecks: ValidityEvidence | DrawableEvidence }) {
-  const { validityChecks } = props;
+export default function ValidityChecks(props: { validityChecks: ValidityEvidence | DrawableEvidence; dateSigned?: Date }) {
+  const { validityChecks, dateSigned } = props;
+  let formattedDateSigned = '';
+  if (dateSigned) {
+    const date = new Date(dateSigned);
+    formattedDateSigned = date.toJSON().slice(0, 19).replace('T', ' ');
```

Comment:
Just use `toJSON()` or `[toISOString()`

![image](https://github.com/criipto/criipto-signatures-validation/assets/699049/79c73ade-c27d-436d-988a-8dfa0e47dae1)

Currently it's not clear what timezone it is.